<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFGWadseekerSites</class>
 <widget class="QWidget" name="CFGWadseekerSites">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListView" name="lstUrls">
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="movement">
      <enum>QListView::Free</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leUrl"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnUrlDefault">
       <property name="toolTip">
        <string>Adds default URLs to the list. No URL that is currently on the list will be removed.</string>
       </property>
       <property name="text">
        <string>Add Defaults</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../media/resources.qrc">
         <normaloff>:/icons/doomseeker_plus.png</normaloff>:/icons/doomseeker_plus.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnUrlAdd">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../media/resources.qrc">
         <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUrlRemove">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../media/resources.qrc">
         <normaloff>:/icons/x.png</normaloff>:/icons/x.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../media/resources.qrc"/>
 </resources>
 <connections/>
</ui>
